inst/REFERENCES.bib

```diff
@@ -0,0 +1,34 @@
+@article{paulodrylands,
+author = {Bernardino, Paulo N. and De Keersmaecker, Wanda and Fensholt, Rasmus and Verbesselt, Jan and Somers, Ben and Horion, Stéphanie},
+title = {Global-scale characterization of turning points in arid and semi-arid ecosystem functioning},
+journal = {Global Ecology and Biogeography},
+volume = {29},
+number = {7},
+pages = {1230-1245},
```

Comment:
1230--1245  ?

Reply:
it checks out https://onlinelibrary.wiley.com/toc/14668238/2020/29/7